### ISSUE TITLE
Bodymovin not rendering certain layers/renders others incorrectly

### ISSUE THREAD

**Issue by ringoditzo** [airbnb/lottie-web#2637]
**Tell us about your environment**

* **Browser and Browser Version:**
* **After Effects Version:**
18.4.1
**What did you do? Please explain the steps you took before you encountered the problem.**
I'm trying to export an animation using bodymovin, because our developer ran into size issues using gif export. But several layers at the beginning of the animation just don't seem to be rendering, and others are rendering weirdly (wrong placement/rotation). 

At the beginning, the character should look like this:
![s1](https://user-images.githubusercontent.com/90471328/132892719-ee36a323-1185-4084-a41f-63c14959ed6d.PNG)
When I export it as a JSON, the character looks like this:
![s2](https://user-images.githubusercontent.com/90471328/132892770-be614024-760b-4841-8d49-63f36cc01034.PNG)

**What did you expect to happen?**

**What actually happened? Please include as much _relevant_ detail as possible.**
Nothing I'm trying seems to help. The layers in question were originally .ai files in a precomp, but I changed them to shapes and put the layers in the parent comp - still didn't work. Would really appreciate help!
**Please provide a download link to the After Effects file that demonstrates the problem.**
https://drive.google.com/file/d/14GWdbwefqcMWUb2zJNkKwVN7kapyv7Cf/view?usp=sharing

**Comment by bodymovin** [comment]
hi, the project says there are 72 files missing, can you do a collect and share it?

**Comment by ringoditzo** [comment]
Shoot sorry - I'm kind of new to this. I tried to package the file at this link: https://drive.google.com/file/d/1_jF6JpKk-lZGyIRt9o-dw9WzZZ_4ue4_/view?usp=sharing

Please let me know if that still isn't working for you! Thanks for the speedy response.

**Comment by bodymovin** [comment]
hi, I'm seeing it correctly when exported.
Can you share a link to the animation playing?

**Comment by ringoditzo** [comment]
Yes - here's what I see when it exports... His legs are missing as he's walking in/arm is off. Is there a setting you're using in exporting that produces a different result? https://lottiefiles.com/share/xl6thzyr

**Comment by ringoditzo** [comment]
Hi, I just wanted to check in - is this something you can help me resolve? Thank you

**Comment by bodymovin** [comment]
The project doesn't seem to have the same values as the animation in lottiefiles. Can you try updating bodymovin version and exporting the same project you shared above?
